src/Drupal/Driver/DrushDriver.php

```diff
@@ -189,11 +189,27 @@ public function clearStaticCaches() {
   }
 
   /**
+   * Decodes JSON output returned by Drush.
+   *
+   * It will clean up any junk that may have appeared before the JSON.
+   *
+   * @param string $output
+   *   The output from Drush.
+   * @return object
+   *   The decoded JSON object.
+   */
+  protected function decodeJsonOutput($output) {
+    // Remove anything before the first '{'.
+    $output = preg_replace('/^[^\{]*/', '', $output);
```

Comment:
Here is a crazy regex that strips in one pass (tested):

`$output = preg_replace('/(^[^\{]*|[^\}]*$)/', '', $output);`

Doing it in two lines might be a lot clearer, though.